packages/basic/lib/base.js

```diff
@@ -68,12 +67,18 @@ export class BaseStackable extends EventEmitter {
       setBasePath: this.setBasePath.bind(this),
       runtimeBasePath: this.runtimeConfig?.basePath ?? null,
       invalidateHttpCache: this.#invalidateHttpCache.bind(this),
-      prometheus: { client, registry: this.metricsRegistry },
       setCustomHealthCheck: this.setCustomHealthCheck.bind(this),
       setCustomReadinessCheck: this.setCustomReadinessCheck.bind(this),
       notifyConfig: this.notifyConfig.bind(this),
       logger: this.logger
     })
+
+    if (globalThis.platformatic.prometheus) {
+      this.metricsRegistry = globalThis.platformatic.prometheus.registry
+    } else {
+      this.metricsRegistry = new client.Registry()
+      this.registerGlobals({ prometheus: { client, registry: this.metricsRegistry } })
+    }
```

Comment:
Why? If there is no global prometheus, there is no metrics globally installed

Reply:
The else branch was the original behavior.
I'm making sure that if anything setup a global registry (like some tests in our codebase) we picked it up.
Note that this was not needed before as plt-basic and plt-service behaved differently, but now we only have a single implementation.